File: Iniciante/1010.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    int cp1, cp2, np1, np2;
    double v1, v2;
    cin >> cp1 >> np1 >> v1;
    cin >> cp2 >> np2 >> v2;
    std::cout << std::fixed << std::setprecision(2);
    cout << "VALOR A PAGAR: R$ " << (np1 * v1 + np2 * v2) << endl;
    return 0;
}

File: Iniciante/C#/1010.cs
using System;
class URI {
    static void Main(string[] args) { 
        string[] pecas1 = Console.ReadLine().Split(' ');
        int cp1 = int.Parse(pecas1[0]);
        int np1 = int.Parse(pecas1[1]);
        double v1 = double.Parse(pecas1[2]);
        
        string[] pecas2 = Console.ReadLine().Split(' ');
        int cp2 = int.Parse(pecas2[0]);
        int np2 = int.Parse(pecas2[1]);
        double v2 = double.Parse(pecas2[2]);
        
        Console.WriteLine("VALOR A PAGAR: R$ " + $"{(np1 * v1 + np2 * v2):0.00}");
    }
}